diffkemp/simpll/ModuleComparator.cpp

```diff
@@ -82,20 +89,30 @@ void ModuleComparator::compareFunctions(Function *FirstFun,
         }
 
         DEBUG_WITH_TYPE(
-                DEBUG_SIMPLL, decreaseDebugIndentLevel();
-                if (ComparedFuns.at({FirstFun, SecondFun}).kind
-                    == Result::EQUAL) {
-                    dbgs() << getDebugIndent() << "Declarations with matching "
-                           << "names, assuming they are equal\n";
-                } else if (ComparedFuns.at({FirstFun, SecondFun}).kind
-                           == Result::NOT_EQUAL) {
-                    dbgs() << getDebugIndent()
-                           << "Declarations without matching "
-                           << "names, assuming they are not equal\n";
-                });
+            DEBUG_SIMPLL,
+
+            decreaseDebugIndentLevel();
+            switch(ComparedFuns.at({FirstFun, SecondFun}).kind) {
+            case Result::EQUAL:
+                dbgs() << "declaration, names are "
+                       << Color::makeGreen("equal")
+                       << " }\n";
+                break;
```

Comment:
I just realized that this case is useless as this code path never sets the `EQUAL` result (uses `ASSUMED_EQUAL` instead). Remove this case and use green colour for `ASSUMED_EQUAL`.